### ISSUE TITLE
Navigation pane (3 line menu)

### ISSUE THREAD

**Issue by garfield227** [ldtteam/MinecoloniesWiki#357]
Wiki 
list item links to:
https://wiki.minecolonies.com/

instead of :

https://ldtteam.github.io/MinecoloniesWiki/
where I was directed by H3lay
...
there are more but I believe that is a big one.
the other I see is that the wiki mentions that there is a button to edit "at the top" but I wasn't seeing that... maybe due to not being logged into github? I will look again... Nope, not seeing an edit option. I can, and will start compiling data for the project as I have time, but it appears there is currently no way to edit the building pages currently (from what I can see) Sorry for such a long post, just trying to do what I can to help out where I can.

**Comment by garfield227** [comment]
found the edit button - didn't see it due to a window not being maximized -live and learn. edits to commence.

**Comment by ravenbuilder934** [comment]
What do you mean by wiki list item?

**Comment by garfield227** [comment]
When the page is narrow, a three line menu appears, similar to a navigation
aid. The link to the wiki in that, is what I was referring to. I maximized
the window, and found the edit page button. I have since included a
suggestion for an edit, but the "main navigation" at the top level still
has what I would call a broken link, back to the old wiki. Maybe this is in
a css page, and used throughout the site? I gave a link that I found, and
the main GitHub link h3lay referred me to, I have not worked with css
before, so maybe that will help point someone in the right direction.

I plan on working on a few of the pages for the guys, eventually all of
them, as I have time, to help add some screenshots, as well as xyz data,
and such for those of us who have had issues with building placement,
etc... Hopefully this will be helpful.

**Comment by lunadragnstar** [comment]
Right now our regular wiki site is down, which is why we send people to the https://ldtteam.github.io/MinecoloniesWiki/ site. As soon as it is back up it should all work fine.

**Comment by lunadragnstar** [comment]
We are currently working on all the pages, Raven has been updating, and I have been trying to get new content added. However, I have multiple jobs and can't spend all my time working on the wiki.

**Comment by garfield227** [comment]
No worries at all, I was just looking for an easy first item to edit /help
on and thought that would be a good thing to update, as practice to get
used to GitHub and the page editor...

Thanks for the quick response, I understand fully that this is a volunteer
type project, and once I found the mod, I have been wanting to provide some
help when I have the time.

I am looking forward to trying to assist with documentation, and hopefully
some screen shots, to help others learn to love what I have found in the
mod, as I placed a builder and town hall too close together, and if I tried
any updates to level up the town hall, it would demolish part of the
builder hut, and vice versa...oopsie...lol. time for a new world...

Thanks again folks for all the hard work you have already done to get this
mod where it is today...I really appreciate it, and am looking forward
helping out during my spare time.
